runtime/java.go

```diff
@@ -52,23 +51,16 @@ func (conf javaRuntime) Validate() error {
 	return conf.CommonRuntime.Validate(inherit)
 }
 func (conf javaRuntime) GetBootCmd(cmdConfs map[string]*CmdConfig, env map[string]string) (string, error) {
-	cmd := fmt.Sprintf("java.so %s io.osv.isolated.MultiJarLoader -mains /etc/javamains", conf.GetJvmArgs())
-	return conf.CommonRuntime.BuildBootCmd(cmd, cmdConfs, env)
-}
-func (conf javaRuntime) OnCollect(targetPath string) error {
-	// Check if /etc folder is already available. This is where we are going to store
-	// Java launch definition.
-	etcDir := filepath.Join(targetPath, "etc")
-	if _, err := os.Stat(etcDir); os.IsNotExist(err) {
-		os.MkdirAll(etcDir, 0777)
-	}
-
-	err := ioutil.WriteFile(filepath.Join(etcDir, "javamains"), []byte(conf.GetCommandLine()), 0644)
-	if err != nil {
-		return err
-	}
-
-	return nil
+	conf.Base = "openjdk8-zulu-compact1:java"
```

Comment:
Do you think we should allow setting the option for the base requirement? From what I remember, compact1 profile may not suffice in some cases (Spark).